server/src/util/test-utils/seed.ts

```diff
@@ -1,10 +1,36 @@
+import { getManager } from 'typeorm';
+import database from '../../sqldb/sqldb';
```

Comment:
I know this is mostly copy/paste but your IDE doesn't seem to warn you of lint issues:

```
 npx eslint src/util/test-utils/seed.ts

/Users/work/dev/workspace-dds/covid19-reports/server/src/util/test-utils/seed.ts
    2:8  error  'database' is defined but never used                                                      @typescript-eslint/no-unused-vars
   29:1  error  'dotenv' should be listed in the project's dependencies. Run 'npm i -S dotenv' to add it  import/no-extraneous-dependencies
   60:1  error  Unexpected tab character                                                                  no-tabs
   60:1  error  Expected indentation of 2 spaces but found 1 tab                                          @typescript-eslint/indent
   67:3  error  Expected exception block, space or tab after '//' in comment                              spaced-comment
  147:3  error  Expected exception block, space or tab after '//' in comment                              spaced-comment
```

Reply:
I will check check, I had to re-install some extensions to my IDE and perhaps this got missed...